backend/src/utils/config.ts

```diff
@@ -135,6 +135,10 @@ export interface Config {
       text: string
       startTimestamp: string
     }
+
+    avScanning: {
+      enabled: boolean
```

Comment:
You need to remove the existing `avScanning.enabled` configuration, as they're now duplicated.